dashboard/src/pages/TreeDetails/TreeDetailsFilter.tsx

```diff
@@ -39,14 +40,18 @@ interface ITreeDetailsFilter {
 }
 
 const filterFieldMap = {
-  config_name: 'configs',
-  architecture: 'archs',
-  compiler: 'compilers',
-  bootStatus: 'bootStatus',
-  testStatus: 'testStatus',
-  buildStatus: 'buildStatus',
-  'duration_[gte]': 'duration_min',
-  'duration_[lte]': 'duration_max',
+  'treeDetails.config_name': 'configs',
+  'treeDetails.architecture': 'archs',
+  'treeDetails.compiler': 'compilers',
+  'treeDetails.valid': 'buildStatus',
+  'treeDetails.duration_[gte]': 'buildDurationMin',
+  'treeDetails.duration_[lte]': 'buildDurationMax',
+  'boot.status': 'bootStatus',
+  'boot.duration_[gte]': 'bootDurationMin',
+  'boot.duration_[lte]': 'bootDurationMax',
+  'test.status': 'testStatus',
+  'test.duration_[gte]': 'testDurationMin',
+  'test.duration_[lte]': 'testDurationMax',
 } as const satisfies Record<string, TFilterKeys>;
```

Comment:
the string here in the Record< seems a little weird, I don't know exatcly where this comes from but this could be a little more specific